content/javascript/concepts/modules/modules.md

```diff
@@ -0,0 +1,162 @@
+---
+Title: "Modules"
+Subjects:
+  - "Web Development"
+  - "Computer Science"
+Tags: 
+  - "Modules"
+CatalogContent:
+  - "introduction-to-javascript"
+  - "paths/front-end-engineer-career-path"
+---
+
+As our program grows bigger, it may contain many lines of code. Instead of putting everything in a single file, we can use modules to separate codes in separate files as per their functionality. This makes our code organized and easier to maintain.
```

Comment:
Let's try to stay away from using "we" as much as we can in Codecademy Docs:

As our program grows bigger, it may contain many lines of code. Instead of putting everything in a single file, modules can be used to separate codes in separate files as per their functionality. This makes the code organized and easier to maintain.